data-model/src/main/avro/StructuredTrace.avdl

```diff
@@ -86,5 +87,7 @@ protocol StructuredTraceProtocol {
     union { null, Graph } event_event_graph = null;
 
     union { null, Graph } entity_event_graph = null;
+
+    union { null, Timestamps } timestamps = null;
```

Comment:
This is a change in the model. Hoping the corresponding sink jobs have been updated to use the latest version of the avdls

Reply:
@avinashkolluru  is there a wiki / steps to determine list of sink jobs ?